src/generateNumber.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const generateNumber = () => {
+  const randoms = [];
+
+  while (randoms.length < 4) {
+    const randomDigit = Math.floor(Math.random() * 10 + 1);
```

Comment:
I think now you generate numbers from 1 to 10, because we use this formula 
'Math.floor(Math.random() * (max - min + 1) + min)'
So if you want to generate numbers from 1 to 9 (we must generate digits) the formula will be looks as
'Math.floor(Math.random() * 9 + 1)'
but maybe I'm wrong)